src/MarginTrading.AssetService.Services/ProductsService.cs

```diff
@@ -217,7 +221,7 @@ public async Task<Result<ProductsErrorCodes>> DiscontinueBatchAsync(string[] pro
             }
 
             _logger.LogInformation("Trying to discontinue products: {Products}", string.Join(',', productIds));
-            var existing = (await _repository.GetAllAsync(null, productIds)).Value.ToHashSet();
+            var existing = (await _repository.GetAllAsync(null, null, productIds)).Value.ToHashSet();
```

Comment:
Same, makes sense to use named arguments for better readability.

```suggestion
            var existing = (await _repository.GetAllAsync(mdsCodeFilter: null, mdsCodes: null, productIds)).Value.ToHashSet();
```

Reply:
done